src/pkg/selectors/onedrive_test.go

```diff
@@ -211,36 +211,69 @@ func (suite *OneDriveSelectorSuite) TestOneDriveRestore_Reduce() {
 		deets        *details.Details
```

Comment:
should deets (above that I can't comment on directly) be populating ItemRef since that's a fallback in the code?

Reply:
yeah, probably should.